x-pack/metricbeat/module/sql/query/_meta/data_postgres_variables_old.json

```diff
@@ -0,0 +1,363 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "event": {
+        "dataset": "sql.query",
+        "duration": 115000,
+        "module": "sql"
+    },
+    "metricset": {
+        "name": "query",
+        "period": 10000
+    },
+    "service": {
+        "address": "localhost:54470",
+        "type": "sql"
+    },
+    "sql": {
+        "driver": "postgres",
+        "metrics": {
+            "numeric": {
+                "archive_timeout": 0,
```

Comment:
Smaller example here too, please.

Reply:
Ignore this file for now, not used in the external doc.